cmake/find_metasmt.cmake

```diff
@@ -0,0 +1,79 @@
+find_package(metaSMT REQUIRED CONFIG)
+message(STATUS "metaSMT_DIR: ${metaSMT_DIR}")
+list(APPEND KLEE_COMPONENT_EXTRA_INCLUDE_DIRS
+  "${metaSMT_INCLUDE_DIR}" ${metaSMT_INCLUDE_DIRS})
+# FIXME: We should test linking
+list(APPEND KLEE_SOLVER_LIBRARIES ${metaSMT_LIBRARIES})
+
+# THIS IS HORRIBLE. The ${metaSMT_CXXFLAGS} variable
```

Comment:
Okay. I've kept the CMake code as it is for now but feel free to submit a PR to change the behaviour once this gets merged into master.